pkg/store/store.go

```diff
@@ -140,9 +141,17 @@ type relationMaps struct {
 
 func (s *atomixStore) Create(ctx context.Context, object *topoapi.Object) error {
 	// If an object is a relation and its ID is empty, build one.
-	if object.ID == "" && object.Type == topoapi.Object_RELATION {
-		relation := object.GetRelation()
-		object.ID = topoapi.RelationID(relation.SrcEntityID, relation.KindID, relation.TgtEntityID)
+	if object.Type == topoapi.Object_RELATION {
+		if object.ID == "" {
+			relation := object.GetRelation()
```

Comment:
So here we can just inject the "uuid:..." instead of what we're doing now, so we can probably move the UUID generation to the top of the method.